### backup

Outdated language strings?

```
FAILED tests/components/backup/test_sensors.py::test_sensors - AssertionError: assert [+ received] == [- snapshot]
    EntityRegistryEntrySnapshot({
     ...
      'original_icon': None,
  -   'original_name': 'Backup Manager State',
  +   'original_name': 'Backup Manager state',
      'platform': 'backup',
     ...
    })
```

### bmw_connected_drive

Outdated language strings?

```
FAILED tests/components/bmw_connected_drive/test_select.py::test_entity_state_attrs - AssertionError: assert [+ received] == [- snapshot]
    EntityRegistryEntrySnapshot({
     ...
      'original_icon': None,
  -   'original_name': 'AC Charging Limit',
  +   'original_name': 'AC charging limit',
      'platform': 'bmw_connected_drive',
     ...
    })
```

### conversation

No idea

```
FAILED tests/components/conversation/test_default_agent.py::test_error_no_device_on_floor - AssertionError: assert 'Sorry, I am not aware of any area called ground floor' == 'Sorry, I am not aware of any device called missing entity on ground floor'
  
  - Sorry, I am not aware of any device called missing entity on ground floor
  ?                              ^ ^^^^        ------------------
  + Sorry, I am not aware of any area called ground floor
  ?                              ^^ ^
```

### environment_canada

API Breakage. 0.10.1 on git differs from 0.10.1 on pypi.

```pytb
ImportError while loading conftest '/build/source/tests/components/environment_canada/conftest.py'.
tests/components/environment_canada/__init__.py:6: in <module>
    from homeassistant.components.environment_canada.const import CONF_STATION, DOMAIN
homeassistant/components/environment_canada/__init__.py:13: in <module>
    from .coordinator import ECConfigEntry, ECDataUpdateCoordinator, ECRuntimeData
homeassistant/components/environment_canada/coordinator.py:10: in <module>
    from env_canada import ECAirQuality, ECRadar, ECWeather, ECWeatherUpdateFailed, ec_exc
E   ImportError: cannot import name 'ECWeatherUpdateFailed' from 'env_canada' (/nix/store/z1dghhs3706a6iw2pb6k5ah780dcnv36-python3.13-env-canada-0.10.1/lib/python3.13/site-packages/env_canada/__init__.py)
```

### google

flaky

### google_travel_time

flaky

### homeassistant_hardware

missing otbr dependencies

### matter
snapshot failure

```
    def snapshot_matter_entities(
        hass: HomeAssistant,
        entity_registry: er.EntityRegistry,
        snapshot: SnapshotAssertion,
        platform: Platform,
    ) -> None:
        """Snapshot Matter entities."""
        entities = hass.states.async_all(platform)
        for entity_state in entities:
            entity_entry = entity_registry.async_get(entity_state.entity_id)
>           assert entity_entry == snapshot(name=f"{entity_entry.entity_id}-entry")
E           AssertionError
```

### nest

flaky

### rainforest_eagle

two state objects are `None` for some reason

```
_______________________________ test_sensors_100 _______________________________

hass = <HomeAssistant RUNNING>
setup_rainforest_100 = <MagicMock name='Eagle100Reader' id='140737180905024'>

    async def test_sensors_100(hass: HomeAssistant, setup_rainforest_100) -> None
        """Test the sensors."""
        assert len(hass.states.async_all()) == 3
    
        demand = hass.states.get("sensor.eagle_100_meter_power_demand")
>       assert demand is not None
E       assert None is not None

demand     = None
hass       = <HomeAssistant RUNNING>
setup_rainforest_100 = <MagicMock name='Eagle100Reader' id='140737180905024'>

tests/components/rainforest_eagle/test_sensor.py:47: AssertionError
```

### rainforest_raven

```
        for entity_entry in entity_entries:
>           assert entity_entry == snapshot(name=f"{entity_entry.entity_id}-entry")
E           AssertionError: assert [+ received] == [- snapshot]
E             Snapshot 'test_sensors[sensor.raven_device_signal_strength-entry]' does not exist!
E             + EntityRegistryEntrySnapshot({
E             +   'aliases': set({
E             +   }),
E             +   'area_id': None,
E             +   'capabilities': dict({
E             +     'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>,
E             +   }),
E             +   'config_entry_id': <ANY>,
E             +   'config_subentry_id': <ANY>,
E             +   'device_class': None,
E             +   'device_id': <ANY>,
E             +   'disabled_by': None,
E             +   'domain': 'sensor',
E             +   'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
E             +   'entity_id': 'sensor.raven_device_signal_strength',
E             +   'has_entity_name': True,
E             +   'hidden_by': None,
E             +   'icon': None,
E             +   'id': <ANY>,
E             +   'labels': set({
E             +   }),
E             +   'name': None,
E             +   'options': dict({
E             +   }),
E             +   'original_device_class': None,
E             +   'original_icon': None,
E             +   'original_name': 'Signal strength',
E             +   'platform': 'rainforest_raven',
E             +   'previous_unique_id': None,
E             +   'supported_features': 0,
E             +   'translation_key': 'signal_strength',
E             +   'unique_id': 'abcdef0123456789.NetworkInfo.link_strength',
E             +   'unit_of_measurement': '%',
E             + })
```

### roku

language strings

```
FAILED tests/components/roku/test_binary_sensor.py::test_roku_binary_sensors - AssertionError: assert 'My Roku 3 Supports Ethernet' == 'My Roku 3 Supports ethernet'
  
  - My Roku 3 Supports ethernet
  ?                    ^
  + My Roku 3 Supports Ethernet
  ?                    ^
FAILED tests/components/roku/test_binary_sensor.py::test_rokutv_binary_sensors[roku/rokutv-7820x.json] - assert '58" Onn Roku TV Supports Ethernet' == '58" Onn Roku TV Supports ethernet'
  
  - 58" Onn Roku TV Supports ethernet
  ?                          ^
  + 58" Onn Roku TV Supports Ethernet
  ?                          ^
```

## sensor

missing translation strings

```
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units0-attributes0-W-kW-GW, MW, TW, W, kW, mW] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units1-attributes1-W-kW-GW, MW, TW, W, kW, mW] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units2-attributes2-\xb0F-K-K, \xb0C, \xb0F] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units3-attributes3-\xb0C-K-K, \xb0C, \xb0F] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units4-attributes4-psi-bar-Pa, bar, cbar, hPa, inHg, kPa, mbar, mmHg, psi] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_unit_change_convertible[units5-attributes5-Pa-bar-Pa, bar, cbar, hPa, inHg, kPa, mbar, mmHg, psi] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units0-attributes0-W-kW-GW, MW, TW, W, kW, mW] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units1-attributes1-W-kW-GW, MW, TW, W, kW, mW] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units2-attributes2-\xb0F-K-K, \xb0C, \xb0F] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units3-attributes3-\xb0C-K-K, \xb0C, \xb0F] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units4-attributes4-psi-bar-Pa, bar, cbar, hPa, inHg, kPa, mbar, mmHg, psi] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units5-attributes5-Pa-bar-Pa, bar, cbar, hPa, inHg, kPa, mbar, mmHg, psi] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_device_class[units6-attributes6-%-None-%, <None>] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_state_class_removed[units0-attributes0-W] - Failed: Translation not found for sensor: `issues.state_class_removed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_state_class_removed_issue_cleaned_up[units0-attributes0-W] - Failed: Translation not found for sensor: `issues.state_class_removed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_conversion[attributes0-cats-dogs] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_no_conversion[attributes1-cats-dogs] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_unit_change_equivalent_units_2[attributes0-m\xb3-m3-CCF, L, fl. oz., ft\xb3, gal, mL, m\xb3] - Failed: Translation not found for sensor: `issues.units_changed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_update_statistics_issues[units0-attributes0-W] - Failed: Translation not found for sensor: `issues.state_class_removed.description`. Please add to homeassistant/components/sensor/strings.json
ERROR tests/components/sensor/test_recorder.py::test_validate_statistics_mean_type_changed - Failed: Translation not found for sensor: `issues.mean_type_changed.description`. Please add to homeassistant/components/sensor/strings.json
```

### stream

flaky


### voip

flaky

--- a/pkgs/development/python-modules/bellows/default.nix
+++ b/pkgs/development/python-modules/bellows/default.nix
@@ -5,8 +5,6 @@
   click,
   click-log,
   fetchFromGitHub,
-  pure-pcapy3,
-  pyserial-asyncio,
   pytest-asyncio,
   pytest-timeout,
   pytestCheckHook,
@@ -18,16 +16,14 @@
 
 buildPythonPackage rec {
   pname = "bellows";
-  version = "0.44.0";
+  version = "0.44.1";
   pyproject = true;
-
-  disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "zigpy";
     repo = "bellows";
     tag = version;
-    hash = "sha256-9CuZGorTqay09ZHrUDoorYew8vvDV6pyxMuik7/QKJ4=";
+    hash = "sha256-LxwKS4jh/ehYGxhU4lpmzXBWGacLtjq6c/lhBybLaRE=";
   };
 
   postPatch = ''
@@ -41,8 +37,6 @@ buildPythonPackage rec {
   dependencies = [
     click
     click-log
-    pure-pcapy3
-    pyserial-asyncio
     voluptuous
     zigpy
   ] ++ lib.optionals (pythonOlder "3.11") [ async-timeout ];
@@ -58,7 +52,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python module to implement EZSP for EmberZNet devices";
     homepage = "https://github.com/zigpy/bellows";
-    changelog = "https://github.com/zigpy/bellows/releases/tag/${version}";
+    changelog = "https://github.com/zigpy/bellows/releases/tag/${src.tag}";
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ mvnetbiz ];
     mainProgram = "bellows";

--- a/pkgs/development/python-modules/zha/default.nix
+++ b/pkgs/development/python-modules/zha/default.nix
@@ -27,7 +27,7 @@
 
 buildPythonPackage rec {
   pname = "zha";
-  version = "0.0.55";
+  version = "0.0.56";
   pyproject = true;
 
   disabled = pythonOlder "3.12";
@@ -36,7 +36,7 @@ buildPythonPackage rec {
     owner = "zigpy";
     repo = "zha";
     tag = version;
-    hash = "sha256-LiHJk7xVdIqNl6BTfwlch3jqQm/MvbrKSExhOR5KD0c=";
+    hash = "sha256-k6Zegn7YUw5Wueb/8Z1RJ7teRMQ6Owdm8obM6XHI/aY=";
   };
 
   postPatch = ''

--- a/pkgs/development/python-modules/zigpy/default.nix
+++ b/pkgs/development/python-modules/zigpy/default.nix
@@ -12,9 +12,7 @@
   fetchFromGitHub,
   freezegun,
   frozendict,
-  importlib-resources,
   jsonschema,
-  pycryptodome,
   pyserial-asyncio,
   pytest-asyncio,
   pytest-timeout,
@@ -27,16 +25,14 @@
 
 buildPythonPackage rec {
   pname = "zigpy";
-  version = "0.78.0";
+  version = "0.78.1";
   pyproject = true;
-
-  disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     owner = "zigpy";
     repo = "zigpy";
     tag = version;
-    hash = "sha256-7ckpg1ukuASWtTdQn/P0KfXaMo5l2NyB9alCXeTarEU=";
+    hash = "sha256-b+4KqcswAKUNJb4e450VwmAR0mca9ApW4n+kif7BR7o=";
   };
 
   postPatch = ''
@@ -47,22 +43,18 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
-  dependencies =
-    [
-      attrs
-      aiohttp
-      aiosqlite
-      crccheck
-      cryptography
-      frozendict
-      jsonschema
-      pyserial-asyncio
-      typing-extensions
-      pycryptodome
-      voluptuous
-    ]
-    ++ lib.optionals (pythonOlder "3.9") [ importlib-resources ]
-    ++ lib.optionals (pythonOlder "3.11") [ async-timeout ];
+  dependencies = [
+    attrs
+    aiohttp
+    aiosqlite
+    crccheck
+    cryptography
+    frozendict
+    jsonschema
+    pyserial-asyncio
+    typing-extensions
+    voluptuous
+  ] ++ lib.optionals (pythonOlder "3.11") [ async-timeout ];
 
   nativeCheckInputs = [
     aioresponses

